exercises/nucleotide-count/tests/Tests.elm

```diff
@@ -10,16 +10,19 @@ tests =
     describe "NucleotideCount"
         [ test "the solution is for the correct version of the test" <|
             \() -> Expect.equal 2 version
```

Comment:
We should probably unskip the first actual test. The version check should automatically pass :)